mobile.css

```diff
@@ -646,7 +646,7 @@ span {
 .error {
   color: #da2c2c;
   font-weight: 400;
-  font-size: 1rem;
+  font-size: 1.6rem;
```

Comment:
- Update your `README` file.

Reply:
@carlos-ssh what in particular should I update?

Reply:
You can get a copy provided by Microverse [here](https://github.com/microverseinc/readme-template).

Try to fill all the best.. Remember that this is the first impression on each project. looking to impress recruiters or people without technical knowledge. 

Remember that you can reutilize your `README` for your next projects just changing simple things.